README.md

```diff
@@ -33,7 +33,23 @@ The following options will be supported
 
 |Option|Short version|Required|Description|
 |-|-|-|-|
-|`--bucket`||true|The name of the bucket to place the lock file|
-|`--prefix`||true|The prefix to be prepended to the key of the lock file|
+|`--bucket`|||The name of the bucket to place the lock file|
+|`--prefix`|||The prefix to be prepended to the key of the lock file|
 |`--token`|||The token to be used to ensure uniqueness.<br />If not specified, an hash of the executable is used|
```

Comment:
Do we need to specify the region as well?

Reply:
Created issue to discuss this: #3